### PR TITLE
Firmware Update for SH without profile-Module/Example/Testcase

### PR DESCRIPTION
**Pull request by alisha-k-kalladassery**
### Description
Firmware Update for Server Hardware without a server profile

### Issues Resolved
[List any issues this PR will resolve. e.g., Fixes #01]

### Check List
- [x] New functionality includes testing.
  - [ ] All tests pass for Python 2.7+ & 3.4+(`$ tox`).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in the examples (please include helpful comments).
  - [ ] New endpoints supported are updated in the endpoints-support.md file.
- [ ] Changes are documented in the CHANGELOG file.

### CODE REVIEW

File: examples/certificate_rabbitmq.py [hunk 1]
```diff
@@ -52,7 +52,7 @@
     response = oneview_client.certificate_rabbitmq.generate(certificate_ca_signed_client)
     pprint(response)
 except HPEOneViewException as e:
-    print (e.msg)
+    print(e.msg)
     if e.oneview_response:
```
Comment (nabhajit-ray) [inline examples/certificate_rabbitmq.py:56]: Why this file is changed?

File: examples/licenses.py [hunk 2]
```diff
@@ -59,7 +59,7 @@
 pprint(license)
 
 # Delete License by ID
-print ("\n\n   ********** Delete the license by ID:  **********")
+print("\n\n   ********** Delete the license by ID:  **********")
 print(uri)
```
Comment (nabhajit-ray) [inline examples/licenses.py:63]: why this is changed?

File: hpeOneView/resources/servers/server_hardware.py [hunk 3]
```diff
@@ -326,3 +333,68 @@ def get_local_storage(self, ip=None):
...
+        state = self.data["state"]
+        server_profile_uri = self.data["serverProfileUri"]
+        power_state = self.data["powerState"]
+        generation = self.data["mpModel"]
+        license_type = self.data["mpLicenseType"]
+
+        validation_error_list = []
+
+        matches = re.findall("\d+$", generation)
+        if int(matches[0]) < 5:
```
Comment (nabhajit-ray) [inline hpeOneView/resources/servers/server_hardware.py:369]: How it is checking the generation here?

File: examples/certificate_rabbitmq.py (hunk 1 above)
Comment (alisha-k-kalladassery) [reply examples/certificate_rabbitmq.py:56]: flake8 showing syntax issue here

File: hpeOneView/resources/servers/server_hardware.py (hunk 3 above)
Comment (alisha-k-kalladassery) [reply hpeOneView/resources/servers/server_hardware.py:369]: generation is checked from mpModel. Gen10 is of iLO5 or more.

**Review by nabhajit-ray — COMMENTED** [review]
Add changelog file

**Comment by nabhajit-ray** [comment]
> * [ ]  New functionality includes testing.
>   
>   * [ ]  All tests pass for Python 2.7+ & 3.4+(`$ tox`).
> * [ ]  New functionality has been documented in the README if applicable.
>   
>   * [ ]  New functionality has been thoroughly documented in the examples (please include helpful comments).
>   * [ ]  New endpoints supported are updated in the endpoints-support.md file.
> * [ ]  Changes are documented in the CHANGELOG file.

Fill the checklist in the PR.
 New functionality includes testing.
 All tests pass for Python 2.7+ & 3.4+($ tox).
 New functionality has been documented in the README if applicable.
 New functionality has been thoroughly documented in the examples (please include helpful comments).
 New endpoints supported are updated in the endpoints-support.md file.
 Changes are documented in the CHANGELOG file.

**Review by nabhajit-ray — APPROVED** [review]

**Review by AvijitOS — APPROVED** [review]

**Pull request merged**